### ISSUE TITLE
Support TLS

### ISSUE THREAD

**Issue by ypaq** [botsunit/kafe#26]
Am I reading the source code correctly here? Kafe doesn't support TLS?

**Comment by sulphur** [comment]
yes that is correct. No TLS support for now, sorry to disappoint you... maybe ont day there will be one ;)
